src/Components/ThanksScreen.js

```diff
@@ -39,7 +46,7 @@ goBackToMap(role) {
     .then(() =>
       this.goBackToMap()
     )
-    .catch(error => console.log('There is an error :( ', error))
+    .catch(() => this.setModalVisible(!this.state.modalVisible))
```

Comment:
I can't comment on it because it wasn't a change, but line 44 needs to be changed! It can't send the whole of state anymore. compare with my other branch (you can copy paste). It now needs to send only the relevant information. Other than that this all looks good